transfers/client.go

```diff
@@ -0,0 +1,55 @@
+package transfers
+
+import (
+	"github.com/checkout/checkout-sdk-go/client"
+	"github.com/checkout/checkout-sdk-go/common"
+	"github.com/checkout/checkout-sdk-go/configuration"
+)
+
+type Client struct {
+	configuration *configuration.Configuration
+	apiClient     client.HttpClient
+}
+
+func NewClient(configuration *configuration.Configuration, apiClient client.HttpClient) *Client {
+	return &Client{
+		configuration: configuration,
+		apiClient:     apiClient,
+	}
+}
+
+func (c *Client) InitiateTransferOfFounds(request TransferRequest) (*TransferOfFoundsResponse, error) {
```

Comment:
Isn't a name mismatch here `TransferRequest` but we return `TransferOfFoundsResponse`, so either TransferRequest -> TransferResponse or TransferOfFoundsRequest -> TransferOfFoundsResponse

Reply:
Also it's missing the idempotent function here

Reply:
Yes, I'll fix both!